README.md

```diff
@@ -445,8 +445,17 @@ In this case, the plugin container receives the `--port 8080` arguments. Note th
 You can create encrypted ecdsa and bls keys using the cli which will be needed for operator registration and other onchain calls
 
 ```bash
-eigenlayer operator keys create  --key-type ecdsa [keyname]
-eigenlayer operator keys create  --key-type bls [keyname]
+eigenlayer operator keys create --key-type ecdsa [keyname]
+eigenlayer operator keys create --key-type bls [keyname]
+```
+
+### Import keys
+
+You can import existing ecdsa and bls keys using the cli which will be needed for operator registration and other onchain calls
+
+```bash
+eigenlayer operator keys import --key-type ecdsa [keyname] [privatekey]
```

Comment:
Is the `privatekey` the plain string or a file path to plain string or something else? Also what `keyname` is for? It might be good to have an example.

Reply:
It's plain text hex string. I will add that explanation .
and keyname is just the name of the file it will be stored with. let me update the documentation and the command manual .

Reply:
added